docs/tutorial.rst

```diff
@@ -15,7 +15,7 @@ First, let's generate some random data for the tutorial.
 
 ``data`` is a pandas DataFrame.
 It must contain a column for entity identifier named **entity**,
-a column for variant, and one column per kpi/feature.
+a column for variant, and one column per kpi/feature. Each data is constructed per each Statistical Test.
```

Comment:
I find this addition slightly confusing. What is meant by 'each data'?

Reply:
@gb oh yes, my mistake, will change that.